staging/src/k8s.io/component-base/cli/flag/ciphersuites_flag.go

```diff
@@ -53,21 +48,65 @@ var ciphers = map[string]uint16{
 	"TLS_AES_256_GCM_SHA384":                  tls.TLS_AES_256_GCM_SHA384,
 }
 
-func TLSCipherPossibleValues() []string {
```

Comment:
preserve this and return the combination of both lists

Reply:
done, ptal